src/Config/Settings.php

```diff
@@ -2,10 +2,11 @@
 
 namespace CodeIgniter\Settings\Config;
 
+use CodeIgniter\Config\BaseConfig;
 use CodeIgniter\Settings\Handlers\ArrayHandler;
 use CodeIgniter\Settings\Handlers\DatabaseHandler;
 
-class Settings
+class Settings extends BaseConfig
```

Comment:
I'm not sure if there were recursion issues or something why Lonnie didn't originally make this an actual Config class. @lonnieezell you available to comment?